lib/backend/k8s/converter/conversion.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package converter
```

Comment:
Seems weird to have this struct defined here with no fields or methods. Looks like they're defined in another file?

Reply:
I was initially trying to split this out and see if I could define some converters per type and this was the first step towards that. This change however isn't relevant anymore as the "KDD Watch" PR from @robbrockbank refactors this and this change will not be necessary.